### ISSUE TITLE
TypeError: Cannot create property 'id' on number '0'

### ISSUE THREAD

**Issue by zabaat** [mesqueeb/vuex-easy-firestore#74]
I'm inserting a new document and get the error

```
TypeError: Cannot create property 'id' on number '0'
    at Store.insert (index.esm.js?34ed:993)
```

```js
        addMod() {
            const order = this.itemMods.length
            this.$store.dispatch('mod/insert', { order })
             // order = { order: 0 }
        },
```

if I change that by simply adding another key it works.
```js
        addMod() {
            const order = this.itemMods.length
            this.$store.dispatch('mod/insert', { order, required:true })
        },
```

store module (even though it doesn't even make it this far)
```js

export default {
    mod: {
        firestorePath: 'mod/',
        firestoreRefType: 'collection',
        moduleName: 'mod',
        statePropName: 'modList',
        sync: {
            insertHook(updateStore, doc, store) {
                console.log('inserted', doc)
                const defaultItem = {
                    id: doc.id,
                    required: true,
                    label: 'Option Name',
                    min: 1,
                    max: 1,
                    order: doc.order,
                    basePriceOverride: false,
                    type: 'multiple-choice',
                    choices: [
                        {
                            key: '0',
                            label: '',
                            order: 0,
                            p: '0',
                        },
                    ],
                }
                console.log('updating store', defaultItem)
                return updateStore(defaultItem)
            }
        },
    },
}
```

**Comment by mesqueeb** [comment]
@zabaat thanks for this. I have an idea what the big might be. If it's as I think I'll push a fix in a few hours. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mesqueeb** [comment]
@zabaat Thank you so much. Fixed in `v1.16.4`.

`npm i vuex-easy-firestore@latest`
